source/_integrations/xiaomi_miio.markdown

```diff
@@ -491,6 +493,38 @@ Buzzer                  | Turn on/off the buzzer
 Child Lock              | Turn on/off the child lock
 LED                     | Turn on/off the LED
 
+### Air Fresh A1 (dmaker.airfresh.a1)
+
+- Power (on, off)
+- Operation modes (Auto, Sleep, Favorite)
+- Sensor entities
+
+Sensor                          | Description                                                    | Enabled by default
+------------------------------- | -------------------------------------------------------------- | -----------------------
+Carbon Dioxide                  | The current carbon dioxide measured in ppm                     | True
+Dust filter life remaining      | The remaining life of the filter                               | True
+Dust filter life remaining days | Filter usage time in day                                       | True
+PM2.5                           | The current particulate matter 2.5 measured                    | True
+Temperature                     | The current outside temperature measured                       | True
+Control Speed                   | The current motor speed measured in rpm                        | True
+Favorite Speed                  | The favorite motor speed measured in rpm                       | True
+Ptc status                      | The heater status                                              | True
```

Comment:
I'm not sure how common term "ptc" is, but would it be better to have "Heater status" instead in its place?

Reply:
This is Auxiliary heat in the MiHome. Maybe Auxiliary heat status?

Reply:
@rytilahti now this is better?

Reply:
"ptc comes from" - miio protocol :)

But yes, maybe using a better name inside the library is a good idea. Some standardization with other similar devices would be a plus. I'll add a note to myself about this for the future.